src/bindings/jna/plugins-kotlin/regex-example/src/README.md

```diff
@@ -0,0 +1,31 @@
+# Regex-Example Kotlin Plugin
+
+This plugin can be used to validate the values of keys via the `check/ktex-regex` metakey.
+
+It uses the value of the metakey as regex expression to check if newly added values match
+with the stored regex. If not, the plugin raises the `VALIDATION_SYNTACTIC` error code.
+
+#### Example usage
+
+Ensure that newly set values for keys only contain small case letters from the english alphabet:
+
+```sh
```

Comment:
Please describe how to mount the plugin.

Reply:
This is now added with 619e959d63ccaec8c5b89c71eab56a80e6eb78b1.